hpat/tests/test_series.py

```diff
@@ -1595,5 +1595,32 @@ def test_impl():
         np.testing.assert_array_equal(hpat_func(), test_impl())
 
 
+class TestSeriesTake(unittest.TestCase):
```

Comment:
Please follow common rules to create tests in this file

Reply:
No problem!